### PR TITLE
Updating the Helm Charts in Rancher Page

### PR DESCRIPTION
**Pull request by sunilarjun**
Updating the Helm Charts in Rancher page to include information about creating chart repositories through Git/Helm.

Fixes https://github.com/rancher/rancher-docs/issues/1085

### CODE REVIEW

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 1]
```diff
@@ -76,6 +76,24 @@ From the left sidebar select _"Repositories"_.
 
 These items represent Helm repositories, and can be either traditional Helm endpoints which have an index.yaml, or git repositories which will be cloned and can point to a specific branch. In order to use custom charts, simply add your repository here and they will become available in the Charts tab under the name of the repository.
 
+#### Add Custom Git Repositories
+
+Click the `Create` button and be sure to select the target `Git repository containing Helm chart...` to add a custom Git repository that contains your Helm charts or cluster template definitions. The branch name can be specified here if necessary, and if no branch name is provided the main branch will be used by default. At minimum a name and Git repository URL will be needed to finalize creation. An optional field for describing the chart repository is also available. Whenever you add a chart repository to Rancher, it will be available immediately.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:81]:
```suggestion
Click **Create** and select the target, **Git repository containing Helm chart...** to add a custom Git repository that contains your Helm charts or cluster template definitions. 
	
You must enter a name and a Git repository URL. The other fields, including the description, are optional. Enter an alternative branch name if you don't want to pull from the default, `main`.
	
Whenever you add a chart repository to Rancher, it becomes available immediately.
```

UI elements should be in bold, not monotype

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 2]
```diff
@@ -76,6 +76,24 @@ From the left sidebar select _"Repositories"_.
 
 These items represent Helm repositories, and can be either traditional Helm endpoints which have an index.yaml, or git repositories which will be cloned and can point to a specific branch. In order to use custom charts, simply add your repository here and they will become available in the Charts tab under the name of the repository.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:77]:
```suggestion
These items represent Helm repositories, and can be either traditional Helm endpoints which have an index.yaml, or Git repositories which will be cloned and can point to a specific branch. In order to use custom charts, simply add your repository here and they will become available in the Charts tab under the name of the repository.
```

I realize this wasn't added by you, but I wanted to fix the typo.

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 3]
```diff
@@ -76,6 +76,24 @@ From the left sidebar select _"Repositories"_.
...
 
+#### Add Custom Git Repositories
+
+Click the `Create` button and be sure to select the target `Git repository containing Helm chart...` to add a custom Git repository that contains your Helm charts or cluster template definitions. The branch name can be specified here if necessary, and if no branch name is provided the main branch will be used by default. At minimum a name and Git repository URL will be needed to finalize creation. An optional field for describing the chart repository is also available. Whenever you add a chart repository to Rancher, it will be available immediately.
+
+#### Add Custom Helm Chart Repositories
+
+A Helm chart repository is an HTTP server that houses one or more packaged charts. Any HTTP server that can serve YAML files and tar files and can answer GET requests can be used as a repository server.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:85]:
```suggestion
A Helm chart repository is an HTTP server that houses one or more packaged charts. Any HTTP server that can serve YAML and tar files, and answer GET requests, can be used as a repository.
```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 4]
```diff
@@ -76,6 +76,24 @@ From the left sidebar select _"Repositories"_.
...
+
+Helm comes with Helm serve, a built-in package server for developer testing. The Helm team has tested other servers, including Google Cloud Storage with website mode enabled, S3 with website mode enabled or hosting custom chart repository server using open-source projects like [ChartMuseum](https://github.com/helm/chartmuseum).
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:87]:
```suggestion
Helm comes with Helm serve, a built-in package server for developer testing. The Helm team has tested other servers, including Google Cloud Storage with website mode enabled, S3 with website mode enabled, or hosting custom chart repository server using open-source projects such as [ChartMuseum](https://github.com/helm/chartmuseum).
```

I'm not sure how "Helm serve" should be formatted and search results conflate it with the command, `helm serve`. Are they the same thing? This page formats it with backticks: https://zeet.co/marketplace/community/helm-serve

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 5]
```diff
@@ -76,6 +76,24 @@ From the left sidebar select _"Repositories"_.
...
+
+In Rancher, you can add the custom Helm chart repository by clicking the `Create` button and having the target `http(s) URL to an index generated by Helm` selected. You will need to input at minimum a repo name and the index URL address of the chart repository.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:89]:
```suggestion
To add a custom Helm chart repository to Rancher, click **Create** and select **http(s) URL to an index generated by Helm** as the target. Enter a repo name and the index URL address of the chart repository.
```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 6]
```diff
@@ -76,6 +76,24 @@ From the left sidebar select _"Repositories"_.
...
+
+#### Add Private Git/Helm Chart Repositories
+
+Private Git or Helm chart repositories can be added into Rancher using either SSH key credentials or an HTTP basic auth secret, i.e. Username and Password.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:93]:
```suggestion
You can add private Git or Helm chart repositories with SSH key credentials or an HTTP basic auth secret, such as a username and password.
```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md [hunk 7]
```diff
@@ -76,6 +76,24 @@ From the left sidebar select _"Repositories"_.
...
+
+#### Add a Private CA to Repositories
+
 To add a private CA for Helm Chart repositories:
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:97]:
```suggestion
To add a private CA to Helm chart repositories:
```

To match title, also correcting capitalization

**Review by martyav — CHANGES_REQUESTED** [review]
Similar changes should be applied to the versioned pages

**Comment by martyav** [comment]
While trying to figure out how to format "Helm serve" (as in the built-in server), I found out that parts of this page copy directly or almost-directly from https://v2.helm.sh/docs/developing_charts/#chart-repositories without linking to it or indicating that these parts are quotations. I wouldn't flag it if we copied text directly from other SUSE-owned tech docs, but this is a third-party. I left some suggestions on some of the copied text before I realized. I'm going to do another pass to revise it more extensively.

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md (hunk 4 above)
Comment (martyav) [inline docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:87]:
```suggestion
Helm comes with a built-in server. For more information, see the [official Helm docs](https://helm.sh/docs/topics/chart_repository/).
```

File: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md (hunk 3 above)
Comment (martyav) [reply docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md:85]: Lines 18 to 20 in 2.0 versioned docs:

> A Helm chart repository is an HTTP server that houses one or more packaged charts. Any HTTP server that can serve YAML files and tar files and can answer GET requests can be used as a repository server.
>
> Helm comes with Helm serve, a built-in package server for developer testing. The Helm team has tested other servers, including Google Cloud Storage with website mode enabled, S3 with website mode enabled or hosting custom chart repository server using open-source projects like [ChartMuseum](https://github.com/helm/chartmuseum).

Update to:

> You can add your own Helm chart repositories to serve chart packages to Rancher. You can use any HTTP server, as long as the server can respond to GET requests and serve YAML files and tar archives. 
>
> For more information on Helm chart repositories, see the [official Helm docs](https://helm.sh/docs/topics/chart_repository/).

**Comment by sunilarjun** [comment]
@martyav Thanks for the review Marty, I applied the changes overall and thanks for shouting out the necessary rephrasing/source citing!

**Review by martyav — APPROVED** [review]

**Pull request merged**